data/dapps/tunnl.json

```diff
@@ -0,0 +1,6 @@
+{
+  "aliases": {
+    "tunnl": { "chains": ["polygon"], "title": "tunnl" }
```

Comment:
These are not on Defillama, but according to their homepage I assume `tunnl` (lowercased title) is how they want to be branded.